src/plugins/passwd/passwd.c

```diff
@@ -49,6 +49,79 @@ static int validatepwent (struct passwd * pwd)
 	return 0;
 }
 
+#if defined(USE_FGETPWENT_LOCAL)
+// clang-format off
+/* Taken from musl libc
```

Comment:
This is MIT licensed, we need to document it in debian/copyright of the debian branch.